.gitignore

```diff
@@ -0,0 +1,10 @@
+praw.ini
+*.pyc
+.idea/*
+archiver.log
+transcribersofreddit.log
+ocr.log
```

Comment:
These 3 lines might be better abbreviated as *.log, yes? Or is there some use case where we would be committing a file ending in .log?